swarmrl/utils/utils.py

```diff
@@ -327,3 +328,48 @@ def calc_signed_angle_between_directors(
     angle *= jnp.where(orthogonal_dot >= 0, 1, -1)
 
     return angle
+
+
+def create_colloids(
+    n_cols: int,
+    type_: int = 0,
+    center: np.array = np.array([500, 500, 0]),
+    dist: float = 200.0,
+    face_middle: bool = False,
+):
+    """
+    Create a number of colloids in a circle around the center of the box.
+    This method is primarily used for writing tests. It is not used in the
+    actual simulation. The colloids are created in a circle around the center
+    of the box.
+
+    Parameters
+    ----------
+    n_cols : int
+            Number of colloids to create.
+    type_ : int, optional
+            Type of the colloids to create.
+    center : np.ndarray, optional
+            Center of the circle in which the colloids are created.
+    dist : float, optional
+            Distance of the colloids to the center.
+    face_middle : bool, optional
+            If True, the colloids face the center of the circle.
+
+    Returns
+    -------
+    colloids : np.ndarray (n_cols, 3)
```

Comment:
This method returns a list, perhaps you can change the type hint here and add a return type to the function definition.